Microsoft.Toolkit.Uwp.SampleApp/SamplePages/OneDrive Service/OneDriveCode.bind

```diff
@@ -0,0 +1,96 @@
+
+// OneDriveService works for OneDrive Consumer as well as OneDrive For Business (Office 365)
+//
+// Authentication 
+// In order to use the OneDriveService you need to authenticate the user and get an access token
+
+// OneDrive consumer you have two options:
+// 1) If the user is connected to a windows session with a Microsoft Account, the service is able to silently get an access token.
+//    For that you need to associate your application to the Store (Project->Store->Associate App With Store...)
+// 2) Or you have to register your app
+//  - go to https://dev.onedrive.com/app-registration.htm
+//  - When prompted, sign in with your Microsoft account credentials.
+//  - Scroll to the bottom of the page (Live (SDK)), and click Add App
+//  - Enter your app's name and click Create application.
+//  - Copy the Application ID
+//  - Add a platform and select mobile application
+//  - Save
+
+// OneDrive For business you need to register your app from the Azure Management Portal
+// For more information to manualy register your app see go to the following article
+// https://docs.microsoft.com/fr-fr/azure/active-directory/develop/active-directory-authentication-scenarios#basics-of-registering-an-application-in-azure-ad
+ 
+
+// First get the root of your OneDrive
+// By default the service silently connect the current windows user if windows is associate with an MS Account
+var rootFolder= await OneDriveService.Instance.RootFolderAsync();
+
+// if windows is not associate with a MS Account, you need
```

Comment:
tiny typo, should read: "If Windows is not associated with a Microsoft Account, you need"